packages/react-native/ReactAndroid/build.gradle.kts

```diff
@@ -467,6 +469,14 @@ fun enableWarningsAsErrors(): Boolean {
   return value?.toString()?.toBoolean() ?: false
 }
 
+fun getHostIpAddress(): String =
+    NetworkInterface.getNetworkInterfaces().asSequence()
+        .filter { it.isUp && !it.isLoopback }
+        .flatMap { it.inetAddresses.asSequence() }
+        .filter { it is Inet4Address && !it.isLoopbackAddress }
+        .map { it.hostAddress }
+        .firstOrNull() ?: "127.0.0.1"
```

Comment:
This won't work as it is. This IP address is the one used to build React Native (in this case it would be either your local machine or the CI machine that is publishing RN).

You set it as a buildConfigFlag so this value will be static and won't be re-accessed every time

Reply:
I've changed it to use resource value instead of BuildConfig. I think it should work now.